docs/evaluation.md

```diff
@@ -1,82 +1,185 @@
 # Agent Evaluation
 
-!!! warning
+The any-agent evaluation module provides two primitives for evaluating agent traces using LLM-as-a-judge techniques:
```

Comment:
Not a blocker, I would like to comment at this level the 3 alternatives I mentioned (we are missing an explicit mention to "code checks" or whatever).

Maybe a simple table of `Method - Pros - Cons` to better showcase when you might want to use each